pkgs/applications/misc/ArchiSteamFarm/default.nix

```diff
@@ -76,7 +76,7 @@ buildDotnetModule rec {
     description = "Application with primary purpose of idling Steam cards from multiple accounts simultaneously";
     homepage = "https://github.com/JustArchiNET/ArchiSteamFarm";
     license = licenses.asl20;
-    platforms = [ "x86_64-linux" "aarch64-linux" ];
+    platforms = platforms.unix;
```

Comment:
I don't think Unix is correct here, we should take meta.platform from dotnet.

Reply:
Looks like the custom builder automatically defined it, so we can simply drop this line.

Reply:
> I don't think Unix is correct here, we should take meta.platform from dotnet.

With my understanding, https://github.com/NixOS/nixpkgs/blob/nixos-23.05/pkgs/development/compilers/dotnet/default.nix shows that dotnet supports 
```
  runtimeIdentifierMap = {
    "x86_64-linux" = "linux-x64";
    "aarch64-linux" = "linux-arm64";
    "x86_64-darwin" = "osx-x64";
    "aarch64-darwin" = "osx-arm64";
    "x86_64-windows" = "win-x64";
    "i686-windows" = "win-x86";
  };
```


With my understanding, dotnet supports unix and windows(i686+x86_64). However, I haven't tested it on Windows. I'm not sure if I should add windows to ArchiSteamFarm's supported platforms

Reply:
I don't think there's the Windows support in nixpkgs, [the only sources](https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/compilers/dotnet/versions/6.0.nix) being from the four platforms.